Gemfile

```diff
@@ -12,53 +8,52 @@ source 'https://rubygems.org'
 chef_version = ENV.key?('CHEF_VERSION') ? ENV['CHEF_VERSION'] : nil
 
 group :doc do
-  gem 'yard', '~> 0.9.5'
+  gem 'yard'
 end
 
 group :test do
+  gem 'berkshelf', '~> 7.0'
   gem 'rake', '~> 12.0'
-  gem 'berkshelf', '~> 6.0'
 end
 
 group :style do
-  gem 'foodcritic', '~> 10.2.0'
-  gem 'rubocop', '~> 0.39.0'
+  gem 'foodcritic'
+  gem 'rubocop'
```

Comment:
Maybe I wrong, but I think removing the versions from here will make its maintainability more difficult. Mainly due to big rule changes between different foodcritic and rubocop versions. So I prefer to adhere to some specific versions.